wrapacker

```diff
@@ -254,18 +259,20 @@ if [[ $dry_run ]]; then
 	cat <<-EOF
 		$PACKER_BIN build \\
 			-only=$PACKER_PROVIDER \\
+            -var "install_script=$INSTALL_SCRIPT" \\
 			-var "iso_url=$ISO_URL" \\
 			-var "iso_checksum=$ISO_CHECKSUM" \\
-			-var "ssh_timeout=$SSH_TIMEOUT" \\
-			arch-template.json
+			-var "ssh_timeout=$TIMEOUT" \\
```

Comment:
Ahhh, yeah I didn't catch that subtlety when I last merged. I saw that there was still a timeout and I was only paying attention to the fact that my variable was on the next line, I'll fix that up.